modules/reference/appvms/comms.nix

```diff
@@ -87,7 +88,8 @@ in
         applications = lib.mkForce ''
           {
           "element": "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/element-desktop --enable-features=UseOzonePlatform --ozone-platform=wayland",
-          "slack":   "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/chromium --enable-features=UseOzonePlatform --ozone-platform=wayland --app=https://app.slack.com/client ${config.ghaf.givc.idsExtraArgs}"
+          "slack":   "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/chromium --enable-features=UseOzonePlatform --ozone-platform=wayland --app=https://app.slack.com/client ${config.ghaf.givc.idsExtraArgs}",
+          "zoom":    "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/env XDG_CURRENT_DESKTOP=wlroots XDG_SESSION_TYPE=wayland LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${pkgs.zoom-us}/opt/zoom/Qt/lib/ ${pkgs.zoom-us}/opt/zoom/zoom"
```

Comment:
Hey, do you think it would make sense to move `XDG_CURRENT_DESKTOP` and `XDG_SESSION_TYPE` to the `run-waypipe` script defined in `appvm.nix` or perhaps even add them to the appvm's environment so that other apps can use them in the future?